grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -201,16 +201,16 @@ class CycleCountService {
     }
 
     CycleCountDto createCycleCount(CycleCountStartCommand request, Location facility) {
+        // We don't set the status on the cycle count here because it will be done automatically upon save.
         CycleCount newCycleCount = new CycleCount(
                 facility: facility,
                 dateLastRefreshed: new Date()
         )
+
         List<AvailableItem> itemsToSave = determineCycleCountItemsToSave(facility, request.cycleCountRequest.product)
-        newCycleCount.save()
```

Comment:
have you made sure it can be removed? I was running into error that "Not-null property references a transient value - transient instance must be saved before current operation", but I assume the cascading now makes it work, since you introduced the bidirectional association

Reply:
yup with the bidirectional it's not needed anymore